README.md

```diff
@@ -4,34 +4,23 @@
 
 ### Dependencies
 
-SENSR SDK depends on [ZeroMQ](http://zeromq.org/intro:get-the-software) (tested with 4.3.1) and [Protobuf](https://developers.google.com/protocol-buffers/) (tested with 3.6.0).
+SENSR SDK depends on [Websocket](https://en.wikipedia.org/wiki/WebSocket) and [Protobuf](https://developers.google.com/protocol-buffers/) (tested with 3.12.0).
 
 ImGui sample code also depends on [GLFW](https://www.glfw.org/) (optional).
 
 You can install prebuilt binaries or build from source as following.
 
 #### Build Dependencies from Source
 
-ZeroMQ (4.3.1) from source:
+Protobuf (3.11.4) from source:
```

Comment:
I guess we can use binary version instead of installing from source?

Reply:
Actually I never tried. Because in [their document](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation), they recommand cpp user follow this way.